EIPS/eip-5005.md

```diff
@@ -0,0 +1,225 @@
+---
+eip: 5005
+title: Zodiac
+description: A composable design philosophy for DAOs.
+author: Auryn Macmillan (@auryn-macmillan), Kei Kreutler (@keikreutler)
+discussions-to: https://ethereum-magicians.org/t/eip-zodiac-a-composable-design-philosophy-for-daos/8963
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-04-14
+requires: 165
+---
+
+## Abstract
+Zodiac is a philosophy and open standard for composable and interoperable DAO tooling. The key insight is that by separating account (thing that holds tokens, controls systems, and is referenced by others) from the logic that controls it, and making the account agnostic to the logic that controls it, we can create a DAO ecosystem that is much more composable and interoperable, along with being compatible with the vast majority of existing DAO tooling. In short, Zodiac encourages DAOs to use an "avatar" contract (like the Gnosis Safe) as their account, and to then control that avatar with just about any combination of DAO tools and frameworks (Aragon, Colony, Compound/OZ Governor, DAOStack, Moloch, Orca, Tribute, etc).
+
+## Motivation
+Currently, most DAO tools and frameworks are built as somewhat monolithic systems. Wherein account and control logic are coupled, either in the same contract or in a tightly bound system of contracts. This needlessly inhibits the future flexibility of organizations using these tools and encourages platform lock-in via extraordinarily high switching costs.
```

Comment:
I'm not 100% sure, but I think it's more grammatically correct to not start a sentence with "wherein"?

```suggestion
Currently, most DAO tools and frameworks are built as somewhat monolithic systems wherein account and control logic are coupled, either in the same contract or in a tightly bound system of contracts. This needlessly inhibits the future flexibility of organizations using these tools and encourages platform lock-in via extraordinarily high switching costs.
```

Reply:
Yeah, I think that should have been a comma, rather than a period.